src/ctim/generators/events.clj

```diff
@@ -1,3 +1,4 @@
+(comment
```

Comment:
Why this ns is in comment? If not used, please remove it

Reply:
Removed the comment, thanks.

Reply:
Decided to remove as it doesn't even compile.